sample.cps

```diff
@@ -16,7 +16,11 @@
   "components": {
     "sample-core": {
       "type": "interface",
-      "definitions": [ "SAMPLE" ],
+      "definitions": {
+        "*": {
+          "SAMPLE": ""
```

Comment:
This is incorrect. The "old" version was `SAMPLE`, which under the new schema should have a value of `null`. The empty string corresponds to the old schema `SAMPLE=`. (Same later.)

Reply:
So it should be `"SAMPLE": null`?

Reply:
Right.

Reply:
Then we have to update the type. https://cps-org.github.io/cps/schema.html#definitions says it is a mapping to string and not to `string | null`

Reply:
Done: https://github.com/cps-org/cps/pull/70.